redis/src/cluster_routing.rs

```diff
@@ -210,7 +210,39 @@ impl<'a> IntoIterator for &'a SlotAddrs {
     }
 }
 
-pub(crate) type SlotMap = BTreeMap<u16, SlotAddrs>;
+#[derive(Debug, Default)]
+pub(crate) struct SlotMap(BTreeMap<u16, SlotAddrs>);
```

Comment:
IMO this is a self contained change, that can be moved to a separate PR. Merging this early will make the review process for the larger PR faster, since it will reduce the amount of code to review.